ci-operator/config/openshift/build-test-images/openshift-build-test-images-master.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
```

Comment:
while you are referencing this, you're referencing it for a substitution that doesn't actually replace anything in the dockerfile.

that said, the dockerfile uses "FROM busybox" which you probably do not want (since that's a docker.io image), so at a minimum change the dockerfile to reference something on quay.io.  Or fix the substitution below to replace "busybox" in the dockerfile.

Reply:
https://github.com/openshift/build-test-images/pull/4